Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "snowboard"
-version = "1.0.3"
+version = "1.0.4"
```

Comment:
Roll back to 1.0.3 and I'll merge

Reply:
Thanks :heart: for taking the time to review this PR. 

Ok sure :slightly_smiling_face: I will revert the change I thought you follow the [semver](semver.org) releases but apparently I think it is not the case. :slightly_smiling_face: